### ISSUE TITLE
Doublecheck quickstarts after adapter removal

### ISSUE THREAD

**Issue by mposolda** [keycloak/keycloak#28794]
But AFAIK quickstarts are already not using deprecated Java adapters, so hopefully we are good already and this could be just closed (after doublechecking).

**Comment by rmartinc** [comment]
I have been checking teh quickstarts and there is no reference for OIDC adapters and no quickstart uses it.

1. I have tested locally the quickstarts that could be (somehow) related to adapters against the current version of main with no issues at all: `jakarta/jaxrs-resource-server`, `jakarta/servlet-authz-client, `jakarta/servlet-saml-service-provider`, `spring/rest-authz-resource-server`, `js/spa, and `nodejs/resource-server`. No issues and the tests pass OK.
2.  I have checked that all adapter references have been already removed by previous PRs: https://github.com/keycloak/keycloak-quickstarts/pull/403 (removed all OIDC adapter quickstarts) and https://github.com/keycloak/keycloak-quickstarts/pull/441 (removed old SAML adapter quickstart). The only quickstart that uses now the SAML adapter is [servlet-saml-service-provider](https://github.com/keycloak/keycloak-quickstarts/tree/latest/jakarta/servlet-saml-service-provider) via the galleon pack.
3. The quickstarts CI is executed everyday against the nightly build of keycloak. So, if there were references to old OIDc adapters, issues would have been detected long ago.

So I think this is done. The quickstarts have no references to the now removed OIDC adapters because they were already prepared by @pedroigor long ago (I suppose when the adapters were declared deprecated and before they were extended).

So I think this is already done and there is nothing to do here. @mposolda If you don't say anything more I will close this one as already done.
